CODEOWNERS

```diff
@@ -1 +1 @@
-*          @bcipriano @gregdenton @jrray @smith1511 @larsbijl @DiegoTavares @IdrisMiles @splhack
+*          @bcipriano @DiegoTavares @ramonfigueiredo
```

Comment:
Getting a warning for Ramon here.

Reply:
```suggestion
*          @bcipriano @DiegoTavares @ramonfigueiredo 
```

Reply:
Ramon has been added with write permissions to the project, but the error continues to happen, let's see if pushing a new change after the permissions have been changed will solve it.